sdk/src/main/java/com/silanis/esl/sdk/License.java

```diff
@@ -0,0 +1,43 @@
+package com.silanis.esl.sdk;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class License{
+
+    private java.util.Date created;
+    private java.util.Date paidUntil = null;
+    private Plan plan;
+    private String status = "ACTIVE";
+    private List<Transaction> transactions = new ArrayList<Transaction>();
+
+
+    public void setCreated(java.util.Date value) { created = value; }
```

Comment:
Please don't name the parameter as `value`, but name it as the actual parameter name

Reply:
Fixed